src/repository/repository.cpp

```diff
@@ -0,0 +1,94 @@
+#include "repository/repository.h"
+
+#include "repository/schemamanager.h"
+
+#include "util/assert.h"
+#include "util/logger.h"
+
+
+namespace mixxx {
+
+//static
+const QString Repository::kDefaultDatabaseConnectionName = "MIXXX";
+
+// The schema XML is baked into the binary via Qt resources.
+//static
+const QString Repository::kDefaultSchemaFile(":/schema.xml");
+
+//static
+const int Repository::kRequiredSchemaVersion = 27;
+
+namespace {
+
+const Logger kLogger("Repository");
+
+const QString kDatabaseFileName = "mixxxdb.sqlite";
+
+} // anonymous namespace
+
+Repository::Repository(
```

Comment:
Could you explain the name? 

The intermediated name database/mixxxdb was not too bad. 
If a new contributor wants to look how mixxxdb.sqlite is handled, he will likely not search for repository in the first place.